<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="6" passed="1" failed="2" skipped="3">
  <reporter-output>
    <line>
      <![CDATA[Able to capture Screenshot]]>
    </line>
    <line>
      <![CDATA[Able to capture Screenshot]]>
    </line>
  </reporter-output>
  <suite started-at="2020-07-29T15:36:40 IST" name="Suite" finished-at="2020-07-29T15:38:25 IST" duration-ms="104690">
    <groups>
      <group name="testMethods">
        <method signature="TestAppScript.testLoginPage()[pri:0, instance:com.testScript.TestAppScript@46fa7c39]" name="testLoginPage" class="com.testScript.TestAppScript"/>
      </group> <!-- testMethods -->
    </groups>
    <test started-at="2020-07-29T15:36:40 IST" name="Test" finished-at="2020-07-29T15:38:25 IST" duration-ms="104690">
      <class name="com.testScript.TestAppScript">
        <test-method is-config="true" signature="testSetReport()[pri:0, instance:com.testScript.TestAppScript@46fa7c39]" started-at="2020-07-29T15:36:40 IST" name="testSetReport" finished-at="2020-07-29T15:36:40 IST" duration-ms="463" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetReport -->
        <test-method is-config="true" signature="testPreCondintion(java.lang.String)[pri:0, instance:com.testScript.TestAppScript@46fa7c39]" started-at="2020-07-29T15:36:40 IST" name="testPreCondintion" finished-at="2020-07-29T15:37:29 IST" duration-ms="48396" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testPreCondintion -->
        <test-method signature="testLoginPage()[pri:0, instance:com.testScript.TestAppScript@46fa7c39]" started-at="2020-07-29T15:37:29 IST" name="testLoginPage" description="this method is used to login in an application" finished-at="2020-07-29T15:37:43 IST" duration-ms="14209" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLoginPage -->
        <test-method is-config="true" signature="testCaptureScreenshot(org.testng.ITestResult)[pri:0, instance:com.testScript.TestAppScript@46fa7c39]" started-at="2020-07-29T15:37:43 IST" name="testCaptureScreenshot" finished-at="2020-07-29T15:37:44 IST" duration-ms="1375" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testLoginPage status=SUCCESS method=TestAppScript.testLoginPage()[pri:0, instance:com.testScript.TestAppScript@46fa7c39] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCaptureScreenshot -->
        <test-method signature="testVerifyLogo()[pri:1, instance:com.testScript.TestAppScript@46fa7c39]" started-at="2020-07-29T15:37:44 IST" name="testVerifyLogo" description="this method is used to verify logo of application" depends-on-groups="testMethods" finished-at="2020-07-29T15:37:46 IST" duration-ms="1320" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Flipkart] but found [Supermart]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Flipkart] but found [Supermart]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.testScript.AppScript.verifyLogo(AppScript.java:137)
at com.testScript.TestAppScript.testVerifyLogo(TestAppScript.java:54)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testVerifyLogo -->
        <test-method is-config="true" signature="testCaptureScreenshot(org.testng.ITestResult)[pri:0, instance:com.testScript.TestAppScript@46fa7c39]" started-at="2020-07-29T15:37:46 IST" name="testCaptureScreenshot" finished-at="2020-07-29T15:37:47 IST" duration-ms="1816" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testVerifyLogo status=FAILURE method=TestAppScript.testVerifyLogo()[pri:1, instance:com.testScript.TestAppScript@46fa7c39] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[Able to capture Screenshot]]>
            </line>
          </reporter-output>
        </test-method> <!-- testCaptureScreenshot -->
        <test-method signature="testProductSelection()[pri:2, instance:com.testScript.TestAppScript@46fa7c39]" started-at="2020-07-29T15:37:47 IST" name="testProductSelection" description="this method is used to select product from product page" depends-on-groups="testMethods" finished-at="2020-07-29T15:38:19 IST" duration-ms="31132" status="FAIL">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[Unable to locate element: //a[@title='Dettol Original Liquid Hand Wash Refill Hand Wash Bottle']
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'DESKTOP-DELL', ip: '192.168.1.8', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.6'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 78.0.2, javascriptEnabled: true, moz:accessibilityChecks: false, moz:buildID: 20200708170202, moz:geckodriverVersion: 0.26.0, moz:headless: false, moz:processID: 4968, moz:profile: C:\Users\shubh\AppData\Loca..., moz:shutdownTimeout: 60000, moz:useNonSpecCompliantPointerOrigin: false, moz:webdriverClick: true, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, platformVersion: 10.0, rotatable: false, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: 08121760-92f0-4dc8-bbd1-e211eb6e4248
*** Element info: {Using=xpath, value=//a[@title='Dettol Original Liquid Hand Wash Refill Hand Wash Bottle']}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: Unable to locate element: //a[@title='Dettol Original Liquid Hand Wash Refill Hand Wash Bottle']
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'DESKTOP-DELL', ip: '192.168.1.8', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.6'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 78.0.2, javascriptEnabled: true, moz:accessibilityChecks: false, moz:buildID: 20200708170202, moz:geckodriverVersion: 0.26.0, moz:headless: false, moz:processID: 4968, moz:profile: C:\Users\shubh\AppData\Loca..., moz:shutdownTimeout: 60000, moz:useNonSpecCompliantPointerOrigin: false, moz:webdriverClick: true, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, platformVersion: 10.0, rotatable: false, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: 08121760-92f0-4dc8-bbd1-e211eb6e4248
*** Element info: {Using=xpath, value=//a[@title='Dettol Original Liquid Hand Wash Refill Hand Wash Bottle']}
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:323)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:428)
at org.openqa.selenium.By$ByXPath.findElement(By.java:353)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at com.sun.proxy.$Proxy15.getText(Unknown Source)
at com.testScript.AppScript.productSelection(AppScript.java:152)
at com.testScript.TestAppScript.testProductSelection(TestAppScript.java:61)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testProductSelection -->
        <test-method is-config="true" signature="testCaptureScreenshot(org.testng.ITestResult)[pri:0, instance:com.testScript.TestAppScript@46fa7c39]" started-at="2020-07-29T15:38:19 IST" name="testCaptureScreenshot" finished-at="2020-07-29T15:38:19 IST" duration-ms="488" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testProductSelection status=FAILURE method=TestAppScript.testProductSelection()[pri:2, instance:com.testScript.TestAppScript@46fa7c39] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[Able to capture Screenshot]]>
            </line>
          </reporter-output>
        </test-method> <!-- testCaptureScreenshot -->
        <test-method signature="testAddToBasket()[pri:0, instance:com.testScript.TestAppScript@46fa7c39]" started-at="2020-07-29T15:38:19 IST" name="testAddToBasket" description="this method is used to add a selected product to the basket" finished-at="2020-07-29T15:38:19 IST" depends-on-methods="com.testScript.TestAppScript.testProductSelection" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TestAppScript.testAddToBasket()[pri:0, instance:com.testScript.TestAppScript@46fa7c39] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TestAppScript.testAddToBasket()[pri:0, instance:com.testScript.TestAppScript@46fa7c39] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:101)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testAddToBasket -->
        <test-method signature="testComparePrice()[pri:0, instance:com.testScript.TestAppScript@46fa7c39]" started-at="2020-07-29T15:38:19 IST" name="testComparePrice" description="this method is used to compare the price whether it is same or not" finished-at="2020-07-29T15:38:19 IST" depends-on-methods="com.testScript.TestAppScript.testAddToBasket" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TestAppScript.testComparePrice()[pri:0, instance:com.testScript.TestAppScript@46fa7c39] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TestAppScript.testComparePrice()[pri:0, instance:com.testScript.TestAppScript@46fa7c39] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:101)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testComparePrice -->
        <test-method signature="detailsoftestcases()[pri:0, instance:com.testScript.TestAppScript@46fa7c39]" started-at="2020-07-29T15:38:19 IST" name="detailsoftestcases" finished-at="2020-07-29T15:38:19 IST" depends-on-methods="com.testScript.TestAppScript.testComparePrice, com.testScript.TestAppScript.testAddToBasket, com.testScript.TestAppScript.testLoginPage, com.testScript.TestAppScript.testProductSelection" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TestAppScript.detailsoftestcases()[pri:0, instance:com.testScript.TestAppScript@46fa7c39] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TestAppScript.detailsoftestcases()[pri:0, instance:com.testScript.TestAppScript@46fa7c39] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:101)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- detailsoftestcases -->
        <test-method is-config="true" signature="testPreCondition()[pri:0, instance:com.testScript.TestAppScript@46fa7c39]" started-at="2020-07-29T15:38:19 IST" name="testPreCondition" finished-at="2020-07-29T15:38:25 IST" duration-ms="5834" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testPreCondition -->
      </class> <!-- com.testScript.TestAppScript -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
